obs_input/obs_input_reanl_ncep_ozone.txt

```diff
@@ -69,6 +69,8 @@
    amsuabufr      amsua       n16         amsua_n16           0.0     1     1
    amsuabufr      amsua       n17         amsua_n17           0.0     1     1
    amsuabufr      amsua       n18         amsua_n18           0.0     1     1
+!  airsbufr       amsua       aqua        amsua_aqua          0.0     1     1
```

Comment:
I think this line (line 72) should be uncommented for NCEP.  It's my understanding that the NCEP airsbufr file contains AIRS and AMSU-A brightness temperatures. HSB data was also in the file when this sensor worked.  

The comment on line 73 is not needed, is it?  This is the NCEP obs_input file.

If we uncomment line 72, we don't need line 74.

What do you think @ilianagenkova and @ADCollard?